pkgs/development/compilers/swift/wrapper/wrapper.sh

```diff
@@ -156,6 +156,14 @@ if [ -z "${NIX_CC_WRAPPER_FLAGS_SET_@suffixSalt@:-}" ]; then
     source $cc_wrapper/nix-support/add-flags.sh
 fi
 
+# Only add darwin min version flag and set up `DEVELOPER_DIR` if a default darwin min version is set,
+# which is a signal that we're targeting darwin. (Copied from add-flags in libc but tailored for Swift).
+if [ "@darwinMinVersion@" ]; then
+    # Make sure the wrapped Swift compiler can find the overlays in the SDK.
+    NIX_SWIFTFLAGS_COMPILE+=" -I $SDKROOT/usr/lib/swift"
+    NIX_LDFLAGS_@suffixSalt@+=" -L $SDKROOT/usr/lib/swift"
+fi
```

Comment:
Would it make sense to use the hook in `wrapper/default.nix` and then copy the variable from the hook here?

Reply:
Wrappers need to set their variables in the wrapper. Setting the variables in a hook risks not having them set (if it isn’t run) or having them clash with variables or settings outside the wrapper.

This is the equivalent of `add-flags.sh` from cc-wrapper and bintools-wrapper, but everything is consolidated here. Ideally, Swift would use cc-wrapper (with Swift additions) like other compilers do, but that’s way out if scope for this PR.